### ISSUE TITLE
Fenced code block as first element

### ISSUE THREAD

**Issue by michelf** [michelf/php-markdown#40]
From pull #30's comment by Aahan: This:

```
~~~
<ul>
    <li>Fantastic</li>
    <li>Great</li>
</ul>
~~~
```

Shows up like this:

![Code block](http://i.imgur.com/KkP6Y.png)

Looks like when a fenced code block is the first element in the file the HTML block parser misses it.

**Comment by ghost** [comment]
@michelf "Fenced code block as first element" — did you mean, "`ul` as first element in fenced code block"? Just wondering if you misunderstood the issue I reported. Thanks! :)

**Comment by ghost** [comment]
By the way, also happens with:

```
~~~
<!-- TODO: remove optional tags -->
<ul>
    <li>Apples</li>
    <li>Oranges</li>
</ul>
~~~
```

So, `ul` need not actually be the first thing in a code block to break it.

**Comment by michelf** [comment]
I believe I'm actually correct with my diagnostic of the problem. Any fenced code block at the start of the file will be affected... except if it contains nothing that looks like HTML markup, in which case the problem is not exposed.

**Comment by michelf** [comment]
Fixed in commit 6ebb841e7dc2f
